src/pixelator/pna/anndata.py

```diff
@@ -23,16 +23,21 @@ def calculate_antibody_metrics(counts_df):
 
 
 def add_panel_information(adata, panel):
-    """Add extra panel variables to var."""
-    if "uniprot_id" in panel.df.columns:
-        panel_meta_data = panel.df[["marker_id", "uniprot_id", "control", "nuclear"]]
-    else:
-        panel_meta_data = panel.df[["marker_id", "control", "nuclear"]]
-    panel_meta_data = panel_meta_data.set_index("marker_id")
-    panel_meta_data.index = panel_meta_data.index.astype(str)
-    panel_meta_data = panel_meta_data.fillna("no")
-
-    adata.var = adata.var.join(panel_meta_data, how="left")
+    """Add panel data to var."""
+    panel_copy = panel.df.copy()
+    panel_copy = panel_copy.set_index("marker_id")
+    panel_copy.index = panel_copy.index.astype(str)
+    panel_copy = panel_copy.fillna("no")
+
+    adata.var = adata.var.join(panel_copy, how="left")
+
+    adata.uns["panel_header"] = {
```

Comment:
I feel like maybe `panel_metadata` could be a clearer name here? But what do you think?